metadata-ingestion/src/datahub/configuration/git.py

```diff
@@ -43,9 +43,7 @@ class GitReference(ConfigModel):
 
     @validator("repo", pre=True)
     def simplify_repo_url(cls, repo: str) -> str:
-        if repo.startswith("github.com/"):
-            repo = f"https://{repo}"
-        elif repo.startswith("gitlab.com"):
+        if repo.startswith("github.com/") or repo.startswith("gitlab.com"):
```

Comment:
is this related to SIM102 rule? isn't that one ignored?

```
    "SIM102", # Use a single `if` statement instead of nested `if` statements
```

Reply:
Just feels out of control then